resources/js/components/value-display.tsx

```diff
@@ -16,9 +16,11 @@ interface Props {
 export default function ValueDisplay({ description, label, modifiers, value }: Props) {
   return (
     <div className={classWithModifiers(bn, modifiers)}>
-      <div className={`${bn}__label`}>{label}</div>
-      <div className={`${bn}__value`}>{value}</div>
-      {description != null && <div className={`${bn}__description`}>{description}</div>}
+      <div style={{ width: 'max-content' }}>
```

Comment:
this breaks description area (like in kudosu section). just add `align-items: flex-start` to the `value-display` class instead.

Reply:
That makes the hover/click area confined to the rank (bad for `-` and one digits). I've moved the description out of the new div.

Reply:
I see...? Although might as well position it at the bottom left instead (with `data-tooltip-position`).

Also, inlining css is generally avoided.